cmd/director_serve.go

```diff
@@ -54,25 +47,6 @@ func serveDirector( /*cmd*/ *cobra.Command /*args*/, []string) error {
 	}
 	go director.PeriodicCacheReload()
 
-	// The director needs its own private key. If one doesn't exist, this will generate it
-	issuerKeyFile := param.IssuerKey.GetString()
-	err = config.GeneratePrivateKey(issuerKeyFile, elliptic.P256())
-	if err != nil {
-		return errors.Wrap(err, "Failed to generate director private key")
-	}
-
-	// Since director will provide a public JWK for token authentication
-	// We need to initialize public JWK and private JWK at the start
-	_, err = config.GenerateIssuerJWKS()
```

Comment:
Can we add back this line to where it was? I got this error at the start of the director with the changes:

```console
WARNING[2023-11-08T22:09:35Z] Failed to verify director service discovery token: key provider 0 failed: failed to find matching key: no key ID ("kid") specified in token 
```